test/TesterAzureUtils/Streaming/AQStreamFilteringTests.cs

```diff
@@ -32,7 +34,7 @@ public override void Dispose()
             {
                 var deploymentId = this.HostedCluster?.DeploymentId;
                 base.Dispose();
-                AzureQueueStreamProviderUtils.DeleteAllUsedAzureQueues(StreamProvider, deploymentId, TestDefaultConfiguration.DataConnectionString)
+                AzureQueueStreamProviderUtils.DeleteAllUsedAzureQueues(this.HostedCluster.ServiceProvider.GetService<ILoggerFactory>(), StreamProvider, deploymentId, TestDefaultConfiguration.DataConnectionString)
```

Comment:
hmmm, you are accessing the disposed hosted cluster, this should throw... just pass the null logger factory